styling/src/jsMain/kotlin/dev.fritz2.styling/theme/default.kt

```diff
@@ -1346,12 +1346,54 @@ open class DefaultTheme : Theme {
         }
 
         override val variants = object : InputFieldVariants {
+            private val basic: Style<BasicParams> = {
+                lineHeight { normal }
```

Comment:
At least ``lineHeight`` is better suited to the ``sizes`` property.

Proposal:
- add a ``basic`` styling for sizes proerty implementation too and call it from the three sizes